tfx/components/example_gen/csv_example_gen/executor.py

```diff
@@ -31,35 +31,52 @@
 from tfx.utils import io_utils
 
 
+def _int_handler(cell: bytes) -> tf.train.Feature:
+  value_list = []
+  if cell:
+    value_list.append(int(cell))
+  return tf.train.Feature(int64_list=tf.train.Int64List(value=value_list))
+
+def _float_handler(cell: bytes) -> tf.train.Feature:
+  value_list = []
+  if cell:
+    value_list.append(float(cell))
+  return tf.train.Feature(float_list=tf.train.FloatList(value=value_list))
+
+def _bytes_handler(cell: bytes) -> tf.train.Feature:
+  value_list = []
+  if cell:
+    value_list.append(cell)
+  return tf.train.Feature(bytes_list=tf.train.BytesList(value=value_list))
+
 @beam.typehints.with_input_types(List[csv_decoder.CSVCell],
                                  List[csv_decoder.ColumnInfo])
 @beam.typehints.with_output_types(tf.train.Example)
 class _ParsedCsvToTfExample(beam.DoFn):
   """A beam.DoFn to convert a parsed CSV line to a tf.Example."""
 
   def __init__(self):
+    super().__init__()
     self._column_handlers = None
 
   def _process_column_infos(self, column_infos: List[csv_decoder.ColumnInfo]):
     column_handlers = []
     for column_info in column_infos:
       # pylint: disable=g-long-lambda
       if column_info.type == csv_decoder.ColumnType.INT:
-        handler_fn = lambda csv_cell: tf.train.Feature(
-            int64_list=tf.train.Int64List(value=[int(csv_cell)]))
+        handler_fn = _int_handler
       elif column_info.type == csv_decoder.ColumnType.FLOAT:
-        handler_fn = lambda csv_cell: tf.train.Feature(
-            float_list=tf.train.FloatList(value=[float(csv_cell)]))
+        handler_fn = _float_handler
       elif column_info.type == csv_decoder.ColumnType.STRING:
-        handler_fn = lambda csv_cell: tf.train.Feature(
-            bytes_list=tf.train.BytesList(value=[csv_cell]))
+        handler_fn = _bytes_handler
       else:
         handler_fn = None
       column_handlers.append((column_info.name, handler_fn))
 
     self._column_handlers = column_handlers
 
   def process(
+      #pylint: disable=arguments-differ
```

Comment:
this will disable this lint for the entire file, not for the single line

it should be added at end of line 80 or 81 depends on which line has the lint issue

Reply:
i tried putting it at the end (after ':') but it still has the pylint error, let me try putting it at end of line 80

Reply:
i searched and i think because the pylint issue was said to be on line 78, so the disable message can be added to line 78 as well